client/webpack.config.js

```diff
@@ -0,0 +1,99 @@
+var path = require('path');
+
+var webpack           = require('webpack');
+
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var CopyPlugin        = require('copy-webpack-plugin');
+var AssetsPlugin      = require('assets-webpack-plugin');
+var BrowserSyncPlugin = require('browser-sync-webpack-plugin');
+
+var production = process.env.NODE_ENV === 'production';
```

Comment:
The situation with NODE_ENV is already a bit complicated with vm-dev in production or development mode and actual production. Could we avoid adding build-time env to the mix ?

Can we use a --flag or another env name ?